reference_designs/log_hello_world/kernel.cc

```diff
@@ -0,0 +1,34 @@
+#define NOCPP
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#define REL_WRITE 0
+#define REL_READ 1
```

Comment:
Are these used?

Reply:
Nope, these are not needed. Thanks for the catch.